scripts/package/macos-notarize-app.sh

```diff
@@ -30,56 +30,32 @@ ditto -c -k --rsrc --keepParent "$APP_BUNDLE" "${APP_BUNDLE}.zip"
 
 # Submit for notarization
 echo "Submitting $APP_BUNDLE for notarization..."
-RESULT=$(xcrun altool --notarize-app --type osx \
-  --file "${APP_BUNDLE}.zip" \
-  --primary-bundle-id org.musicbrainz.Picard \
-  --username "$APPLE_ID_USER" \
-  --password @env:APPLE_ID_PASSWORD \
-  -itc_provider MetaBrainzFoundationInc \
-  --output-format xml)
+RESULT=$(xcrun notarytool submit \
+  --apple-id "$APPLE_ID_USER" \
+  --team-id "$APPLE_ID_TEAM" \
+  --password "$APPLE_ID_PASSWORD" \
+  --output-format plist \
+  --wait \
+  --timeout 10m \
+  "${APP_BUNDLE}.zip")
 
 if [ $? -ne 0 ]; then
   echo "Submitting $APP_BUNDLE failed:"
   echo "$RESULT"
   exit 1
 fi
 
-REQUEST_UUID=$(echo "$RESULT" | xpath \
-  "//key[normalize-space(text()) = 'RequestUUID']/following-sibling::string[1]/text()" 2> /dev/null)
+STATUS=$(echo "$RESULT" | xpath \
+  "//key[normalize-space(text()) = 'status']/following-sibling::string[1]/text()" 2> /dev/null)
 
-if [ -z "$REQUEST_UUID" ]; then
-  echo "Submitting $APP_BUNDLE failed:"
+if [ "$STATUS" = "Accepted" ]; then
+  echo "Notarization of $APP_BUNDLE succeeded!"
+  break
```

Comment:
```suggestion
  exit 0
```

Reply:
Actually that line needs to be removed completely. A left over from when this was inside the loop. Will fix